Dockerfile

```diff
@@ -1,18 +1,23 @@
-FROM python:3.6.5
+FROM ubuntu:bionic
+
+# ENV PYTHONUNBUFFERED 1
+
```

Comment:
What's going on here? As far as I know, we switched off `ubuntu:bionic` to `python:3.6.5` to improve build times... why is it changing back?